ramls/schemas/exportRequest.json

```diff
@@ -4,15 +4,13 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
-    "fileDefinition": {
-      "description": "File definition",
-      "type": "object",
-      "$ref": "fileDefinition.json"
+    "fileDefinitionId": {
+      "description": "File definition id",
+      "type": "string"
```

Comment:
Sorry I missed this last time. We need to add the pattern here and in job profile ID, so that invalid Strings are not accepted